File: homeworks/sample/java/Apt.java
import java.io.*;
import java.util.*;

public class Apt{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);

		int n = s.nextInt();

		int[] p = new int[n];

		for(int i=0; i<n; i++){
		    p[i] = s.nextInt(); 
		}

		int low = Math.min(p[0],p[1]);
		int sec = Math.max(p[0],p[1]);

		for(int i=2; i<n; i++){
			if(p[i] < low){
			  sec = low;
			  low = p[i];
			}
			else if(p[i] < sec){
			  sec = p[i];
			}
		}

		System.out.println(sec);

		
	}


}

File: homeworks/sample/python/apt.py

n = int(input());

p = [];

for i in range(0,n):
	p.append(int(input()))

low = min(p[0],p[1]);
sec = max(p[0],p[1]);

for i in range(2,n):
	if p[i] < low:
	  sec = low
	  low = p[i]
	elif p[i] < sec:
	  sec = p[i];
	


print(sec)
